combat-trainer.lic

```diff
@@ -1791,7 +1791,35 @@ class TrainerProcess
 
     @pet_count = @skill_map.include?('Locks') ? get_boxes(@pet_box_source).count : 0
     echo("  @pet_count: #{@pet_count}") if $debug_mode_ct
-  end
+	
+	@constellations = get_data('constellations').constellations
+	
+    @have_divination_bones = DRC.get_settings.have_divination_bones
+    
+	@divination_bones_container = DRC.get_settings.divination_bones_storage['container']
+    
+	@divination_bones_tied = DRC.get_settings.divination_bones_storage['tied']
+    
+	@have_telescope = DRC.get_settings.have_telescope
+    
+	@ap_source = DRC.get_settings.astral_plane_training['train_source']
+    
+	@ap_destination = DRC.get_settings.astral_plane_training['train_destination']
+   
+    @prediction_pool_target = DRC.get_settings.astrology_use_full_pools ? 10 : DRC.get_settings.astrology_pool_target
+    
+	@rtr_data = nil
+    
+	@astrology_prediction_skills_magic = DRC.get_settings.astrology_prediction_skills['magic']
+    @astrology_prediction_skills_lore = DRC.get_settings.astrology_prediction_skills['lore']
+    @astrology_prediction_skills_offense = DRC.get_settings.astrology_prediction_skills['offense']
+    @astrology_prediction_skills_defense = DRC.get_settings.astrology_prediction_skills['defense']
+    @astrology_prediction_skills_survival = DRC.get_settings.astrology_prediction_skills['survival']
+	Flags.add('astro_true', 'You feel you have sufficiently pondered your latest observation')
```

Comment:
Im not a big fan of this naming - It should indicate what its doing. Something like 'astrology_ready' would be more understandable

Reply:
'astro_can_observe' as another suggestion

Reply:
Actually I am looking for the astro_true elsewhere in the code and it isnt there.  I would like to keep the idea and change the name.  I think I could add a check that time has passed and the flag is true.  then observe and predict?

Reply:
Even if it is not being used elsewhere, you should use it here. We want to guard against needlessly trying to observe the heavens if we cannot do that.